--- a/src/server.c	2021-04-09 12:37:23.000000000 -0500
+++ b/src/server.c	2021-04-18 06:19:52.414499292 -0500
@@ -11,7 +11,6 @@
 #include <wlr/types/wlr_compositor.h>
 #include <wlr/types/wlr_data_control_v1.h>
 #include <wlr/types/wlr_data_device.h>
-#include <wlr/types/wlr_gtk_primary_selection.h>
 #include <wlr/types/wlr_input_device.h>
 #include <wlr/types/wlr_keyboard.h>
 #include <wlr/types/wlr_output_layout.h>
@@ -610,7 +609,6 @@
 {
   wlr_data_control_manager_v1_create(server->display);
 
-  wlr_gtk_primary_selection_device_manager_create(server->display);
   wlr_primary_selection_v1_device_manager_create(server->display);
 
   server->seat = wlr_seat_create(server->display, "seat0");
