--- a/modules/meson.build	2021-01-14 04:13:16.000000000 -0600
+++ b/modules/meson.build	2021-03-08 10:42:16.043640206 -0600
@@ -5,7 +5,6 @@
 alsa = dependency('alsa')
 udev = dependency('libudev')
 json = dependency('json-c')
-mpd = dependency('libmpdclient')
 xcb_xkb = dependency('xcb-xkb', required: get_option('backend-x11'))
 
 # Module name -> (source-list, dep-list)
@@ -16,8 +15,6 @@
   'clock': [[], []],
   'i3': [['i3-common.c', 'i3-common.h'], [dynlist, json]],
   'label': [[], []],
-  'mpd': [[], [mpd]],
-  'network': [[], []],
   'removables': [[], [dynlist, udev]],
   'script': [[], []],
   'sway-xkb': [['i3-common.c', 'i3-common.h'], [dynlist, json]],
